Makefile

```diff
@@ -25,9 +25,8 @@ clean:
 
 ## Run the development H server locally
 .PHONY: dev
-dev: build/manifest.json .pydeps
-	@bin/hypothesis --dev init
-	@bin/hypothesis devserver
+dev: build/manifest.json
+	tox -e dev
```

Comment:
Once we finish moving everything into tox we're gonna be able to simplify the `Makefile` a lot. The `.pydeps` file isn't needed, nor is `PIP_REQUIRE_VIRTUALENV`, etc.